client/src/locales/en-US.json

```diff
@@ -1542,12 +1542,12 @@
             "information": "Information",
             "startingDate": "Starting date",
             "endingDate": "Ending date",
-            "members": {
-                "title": "License member | Licenses members",
-                "admin": "Administrator member | Administrator members",
+            "users": {
```

Comment:
Nop. Only change the translations, not the key. The code should not be updated, only the translations themselves.
I also don't understand this change. What is a member? Here you're replacing `users` with `members`, in other instances you're replacing `standard` with `member`. Those are different notions.

Reply:
Read the diff in reverse. My bad.